src/renderer/components/ft-shaka-video-player/ft-shaka-video-player.js

```diff
@@ -2382,6 +2382,8 @@ export default defineComponent({
 
       window.addEventListener('beforeunload', stopPowerSaveBlocker)
 
+      container.value.classList.add('no-cursor')
```

Comment:
Could you please add a code comment explaining why this is needed, so that we know it from looking at the code in the future and don't remove it with the reasoning that cursor hiding is shaka-player's job.